routes/quizzes.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const router = express.Router();
+const Quiz = require("../models/quiz");
```

Comment:
```suggestion
const quiz = require("../models/quiz");
```
The convention for JS variable names are that they are only capitalised if they are classes, are instantiable (i.e. you can do `new Quiz()`, or are components

Reply:
I just saw in #80 that we are doing `new Quiz` so we can leave it as `const Quiz`